src/pages/transactions/[txid].page.tsx

```diff
@@ -38,12 +42,24 @@ interface TransactionPageProps {
   transaction?: Transaction;
   vins?: TransactionVin[];
   vouts?: TransactionVout[];
+  mappedEvmTxId?: string | null;
+}
+interface VmmapResult {
+  input: string;
+  type: string;
+  output: string;
+}
+
+enum VmmapTypes {
```

Comment:
can we write VmmapResult and VmmapTypes at one common place ?
similar to, what we did for `governance/enum`